config.json

```diff
@@ -496,6 +496,11 @@
       "difficulty": 1,
       "slug": "forth",
       "topics": []
+    },
+    {
+      "difficulty": 1,
+      "slug": "change",
+      "topics": []
```

Comment:
It would be awesome if you felt open to adding the topics to this in a second commit, and maybe suggesting a difficulty rating?  There's a list of topics here https://github.com/exercism/x-common/blob/master/TOPICS.txt
Don't worry if not, just a thought.